layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -74,43 +74,45 @@ instances:
 ```
 
 ### Monitoring relation metrics for multiple logical databases
-In order to collect relation metrics (such as `postgresql.seq_scans`, `postgresql.dead_rows`, `postgresql.index_rows_read`, and `postgresql.table_size`), the Agent must be configured to connect to each logical database (by default, the Agent only connects to the `postgres` database).
+In order to collect relation metrics (such as `postgresql.seq_scans`, `postgresql.dead_rows`, `postgresql.index_rows_read`, and `postgresql.table_size`), the Agent must be configured to connect to each logical database (by default, the Agent only connects to the `postgres` database). 
```

Comment:
```suggestion
In order to collect relation metrics (such as `postgresql.seq_scans`, `postgresql.dead_rows`, `postgresql.index_rows_read`, and `postgresql.table_size`), `database_autodiscovery` must be enabled because these metrics must be gathered across logical databases on the Postgres instance.
```